src/els_parser.erl

```diff
@@ -259,10 +259,9 @@ attribute(Tree) ->
       end;
     {record, {Record, Fields}} ->
       [poi(Pos, record, Record, Fields)];
-    {type, {type, {Type, _, Args}}} ->
+    {Name, {Name, {Type, _, Args}}} when Name =:= type;
```

Comment:
This is a fix for a bug I found while getting type completions from the `dict` module.